test/common/json/json_loader_test.cc

```diff
@@ -246,7 +246,7 @@ TEST_F(JsonLoaderTest, Hash) {
   ObjectSharedPtr json1 = Factory::loadFromString("{\"value1\": 10.5, \"value2\": -12.3}");
   ObjectSharedPtr json2 = Factory::loadFromString("{\"value2\": -12.3, \"value1\": 10.5}");
   ObjectSharedPtr json3 = Factory::loadFromString("  {  \"value2\":  -12.3, \"value1\":  10.5} ");
-  EXPECT_NE(json1->hash(), json2->hash());
+  EXPECT_EQ(json1->hash(), json2->hash());
```

Comment:
Should these tests be stricter and verify that the hash has a specific value?

Reply:
We could to make it v strict, though changing the underlying container/hash algorithm changes could make this test brittle and require repeated changes

Reply:
Agreed.

possible nit: there are no remaining EXPECT_NE for the hash function.  It may make sense to add something like:

ObjectSharedPtr json4 = Factory::loadFromString("{\"value1\": 10.5}");
EXPECT_EQ(json1->hash(), json4->hash());

Reply:
good point! will add that test assertion